geowebcache/core/src/test/java/org/geowebcache/filter/parameters/FloatParameterFilterTest.java

```diff
@@ -110,10 +110,10 @@ public void testThreshold() throws Exception {
     
     @Test
     public void testToXML() throws Exception {
-        XMLAssert.assertXMLEqual("<floatParameterFilter>\n"+
+        XMLAssert.assertXMLEqual("<floatParameterFilter id=\"1\">\n"+
                 "  <key>TEST</key>\n"+
                 "  <defaultValue>Default</defaultValue>\n"+
-                "  <values>\n"+
+                "  <values id=\"2\">\n"+
                 "    <float>42.0</float>\n"+
                 "    <float>6.283</float>\n"+
                 "    <float>-17.5</float>\n"+
```

Comment:
Having to change every test case example is a bit scary; should check with downstream applications.

@groldan or @tbarsballe do you have an insight?